src/components/SheetTable.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div style="height:100%">
+    <div class="sheetTable" ref="table">
+      <table>
+        <thead>
+          <tr>
+            <th>
+              <label class="cb_container cb_small">
+                <input type="checkbox" />
+                <span class="checkmark"></span>
+              </label>
+            </th>
+            <th>Time</th>
+            <th>Keys</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr
+            v-for="(entry, idx) in instance.timeArr"
+            :key="idx"
+            :class="{onScreen:isWithinTime(entry.t), current:idx===instance.timeArrIdx}"
+          >
+            <td>
+              <label class="cb_container cb_small">
+                <input type="checkbox" />
+                <span class="checkmark"></span>
+              </label>
+            </td>
+            <td>{{entry.t}}</td>
+            <td>{{entry.k==" "?"-":entry.k}}</td>
```

Comment:
```suggestion
            <td>{{entry.k===" "?"-":entry.k}}</td>
```

Or can `entry.k` be "falsey"?

Reply:
Not yet, but will be XD